tests/utils/test_proto_json_utils.py

```diff
@@ -79,22 +80,105 @@ def test_back_compat():
     assert exp_json == in_json
 
 
-def test_parse_tf_serving_input():
+def test_parse_tf_serving_dictionary():
+    def assert_result(result, expected_result):
+        assert result.keys() == expected_result.keys()
+        for key in result:
+            assert (result[key] == expected_result[key]).all()
+
     # instances are correctly aggregated to dict of input name -> tensor
     tfserving_input = {
         "instances": [
-            {"a": "s1", "b": 1, "c": [1, 2, 3]},
-            {"a": "s2", "b": 2, "c": [4, 5, 6]},
-            {"a": "s3", "b": 3, "c": [7, 8, 9]},
+            {"a": "s1", "b": 1.1, "c": [1, 2, 3]},
+            {"a": "s2", "b": 2.2, "c": [4, 5, 6]},
+            {"a": "s3", "b": 3.3, "c": [7, 8, 9]},
         ]
     }
+    # Without Schema
     result = parse_tf_serving_input(tfserving_input)
-    assert (result["a"] == np.array(["s1", "s2", "s3"])).all()
-    assert (result["b"] == np.array([1, 2, 3])).all()
-    assert (result["c"] == np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]])).all()
+    expected_result_no_schema = {}
+    expected_result_no_schema["a"] = np.array(["s1", "s2", "s3"])
+    expected_result_no_schema["b"] = np.array([1.1, 2.2, 3.3], dtype="float64")
+    expected_result_no_schema["c"] = np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]], dtype="int64")
```

Comment:
```suggestion
    expected_result_no_schema = {
       "a": np.array(["s1", "s2", "s3"]),
       "b": np.array([1.1, 2.2, 3.3], dtype="float64"),
       "c": np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]], dtype="int64")
```
nit to keep it consistent with other examples